dev-docs/code-coverage.md

```diff
@@ -31,11 +35,9 @@ UI code coverage can be viewed at:
 
 <https://a2-code-coverage.cd.chef.co/automate-ui/current/>
 
-Past UI code coverage results can be found at `/YYYY-MM-DD` instead of `/current`
-
-To view what previous dates are available, see [Updating Code Coverage Dashboard](#Updating-Code-Coverage-Dashboard).
+Past UI code coverage results can be found at the parent page, https://a2-code-coverage.cd.chef.co/automate-ui/, along with a chart showing changes in the last 90 days.
 
-Currently this is manually updated periodically.
+This is automatically populated each day.
```

Comment:
I think this is populated each time a PR is merged.

Reply:
Thanks; updated.